Taxi/Taxi/Presenter/CellView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  CellView.swift
+//  Taxi
+//
+//  Created by Yosep on 2022/06/09.
+//
+import SwiftUI
+
+struct CellView: View {
+    let party: TaxiParty
+    private let paddingSize: CGFloat = 6
+    var body: some View {
+        VStack(alignment: .leading, spacing: 13) {
+            HStack {
+                MeetTimeView(party: party)
+                Spacer()
+                HStack(spacing: 4) {
+                    Image(systemName: "person.fill")
+                        .font(.system(size: 20))
+                        .foregroundColor(Color(red: 64 / 255, green: 64 / 255, blue: 64 / 255, opacity: 1.0))
+                    Text("\(party.members.count)") // 데이터로 처리 필요
+                        .foregroundColor(Color(red: 64 / 255, green: 64 / 255, blue: 64 / 255, opacity: 1.0))
+                        .font(.custom("Apple SD Gothic Neo", size: 20))
+                    Text("/")
+                        .foregroundColor(Color(red: 206 / 255, green: 206 / 255, blue: 206 / 255, opacity: 1.0))
+                        .font(.custom("Apple SD Gothic Neo", size: 20))
+                    Text("\(party.maxPersonNumber)") // 데이터로 처리 필요
+                        .foregroundColor(Color(red: 206 / 255, green: 206 / 255, blue: 206 / 255, opacity: 1.0))
+                        .font(.custom("Apple SD Gothic Neo", size: 20))
+                }
+            }
```

Comment:
```suggestion
                CurrentPartyMembersView()
```

별도의 struct 혹은 @viewBuilder method 로 정의해서 body를 줄여보는건 어떨까요 ?
각각의 HStack 혹은 VStack이 정리되면, 아래처럼 body의 코드가 심플해지고 특정 영역의 수정이 필요할 때 찾기도 쉽답니다

```swift
VStack {
    HStack {
        MeetTimeView(party: party)
        CurrentPartyMembersView()
    }
    DefatureAndArrivalAreaView()
}

```

Reply:
네 이전에, 사실 위와 같이 Struct 형식으로 코딩을 했었는데, party: party부분을 나머지 Struct에 명시하는 과정중에 error가 나서 다시한번 코드 refactoring해서 수정 시도해보겠습니다.

Reply:
refactoring 완료하였습니다.